suite-common/formatters/src/formatters/prepareNetworkSymbolFormatter.ts

```diff
@@ -12,8 +12,8 @@ export const prepareNetworkSymbolFormatter = (config: FormatterConfig) =>
             const { bitcoinAmountUnit } = config;
             const { areAmountUnitsEnabled = true } = dataContext;
 
-            const { features: networkFeatures, testnet: isTestnet } =
-                networksCompatibility.find(network => network.symbol === symbol) ?? {};
+            // symbol: NetworkSymbol, so TS says getNetwork always returns Network, but fallback just in case (we also have unit tests for this)
```

Comment:
the question is, if we need [those unit tests](https://github.com/trezor/trezor-suite/blob/74d30844a27d67274395ad23e5102c4c3fefe1c1/suite-common/formatters/src/formatters/tests/prepareCryptoAmountFormatter.test.ts#L78-L103).
But I'd rather be conservative, maybe they are there for a reason _(e.g. someone somewhere overrides TS? )_